tests/unit/config/test_logging.py

```diff
@@ -0,0 +1,22 @@
+"""Unit tests for logging configuration."""
+
+import logging
+import os
+
+import pytest
+
+from eq_cir_proxy_service.config import logging_config
+
+
+@pytest.mark.parametrize(
+    "env_value, expected_level",
+    [
+        ("", logging.INFO),  # Default log level
+        ("DEBUG", logging.DEBUG),  # Custom log level
+    ],
+)
+def test_get_log_level(env_value, expected_level):
+    """Test that the logging level is set correctly based on the LOG_LEVEL environment variable."""
```

Comment:
Should this docstring follow the "Should" format instead of the "Test" format, to match the other tests?

Reply:
Updated